### ISSUE TITLE
License

### ISSUE THREAD

**Issue by powerjg** [spwilson2/whimsy#5]
I think people are going to want to see it at the top of every file. IMO this isn't necessary, but...

A related thing we should think about: do we want to keep this in a separate repo (i.e., in gem5/ext and here) or just include it as testlib? I think it should probably be in gem5, not just here. Other thoughts?

**Comment by spwilson2** [comment]
Yeah I figured I would wait till the library was finished before putting the LICENSE in the header. Thanks for the reminder!

I would say to probably keep the framework portion in gem5/ext and then move gem5 specific helpers (those items in whimsy/gem5) into the gem5/tests directory, but that might be too confusing? Not sure.. Hopefully the framework portion wouldn't need to change much as tests are added, but I would expect that more fixtures and test helpers would continue to be written.

**Comment by powerjg** [comment]
I agree with the last sentence. I'm also OK with wherever it ends up.
